tools/cockpit.spec

```diff
@@ -523,6 +523,22 @@ getent group cockpit-wsinstance >/dev/null || groupadd -r cockpit-wsinstance
 getent passwd cockpit-wsinstance >/dev/null || useradd -r -g cockpit-wsinstance -d /nonexisting -s /sbin/nologin -c "User for cockpit-ws instances" cockpit-wsinstance
 
 %post ws
+if firewall-cmd --state; then
```

Comment:
This must happen only on first-time installation, not on upgrades. If the admin changes the firewall policies, updating cockpit must not destroy that. In Debian one can do something like `if [ -z "$2" ]`, where `$2` is the previously installed version. I'm sure RPM has something like that as well.

Reply:
the exact same thing except `$1`